lib/common/formats.js

```diff
@@ -34,29 +34,106 @@ function cssHeader() {
+      var srcIndent = '     ';
+      var asset_path = '../';
+
+      var name = '', weight = '', style = '';
+      var prefix = '@font-face {\n' + tabs + "font-family: '{{NAME}}';\n";
+      var postfix = '}\n';
+      var formats = [];
+
+      Object.keys(properties).forEach(function (key) {
+        var prop = properties[key];
+
+        if(key === 'name' || key === 'weight' || key === 'style') {
+          return;
+        }
+        else {
+          var filepath = prop.split('.');
+          var file_suffix = filepath.pop();
+          filepath = asset_path + filepath.join('.');
+
+          switch (file_suffix) {
+            case 'eot':
+              formats.unshift(tabs + srcIndent + "url('" + filepath + ".eot');\n" + tabs + "src: url('" + filepath + ".eot?#iefix') format('embedded-opentype')");
+              break;
+            case 'svg':
+              formats.push(tabs + srcIndent + "url('" + filepath + ".svg#{{NAME}}') format('svg')");
+              break;
+            case 'ttf':
+              formats.push(tabs + srcIndent + "url('" + filepath + ".ttf') format('truetype')");
+              break;
+            case 'woff':
+            case 'woff2':
+            default:
+              formats.push(tabs + srcIndent + "url('" + filepath + "." + file_suffix + "') format('" + file_suffix + "')");
+              break;
+          }
+        }
+      });
+
+      if(properties.style) {
+        postfix = tabs + 'font-style: ' + properties.style + ';\n' + postfix;
+      }
+      if(properties.weight) {
+        postfix = tabs + 'font-weight: ' + properties.weight + ';\n' + postfix;
+      }
 
-        if (prop.attributes.comment)
-          to_ret = to_ret.concat(' /* ' + prop.attributes.comment + ' */ ');
-        return to_ret;
+      var font_face = prefix + tabs + 'src: ' + formats.join(',\n').trim() + ";\n" + postfix;
+      font_face = font_face.replace(/\{\{NAME\}\}/g, properties.name);
+
+      return font_face;
+    };
+
+    var font_face_arr = [];
+    Object.keys(fontFaces).forEach(function (key) {
+      var fontFace = fontFaces[key];
+
+      if(fontFace) {
+        font_face_arr.push(genSrc(fontFace));
       }
+    });
+    font_face_arr = font_face_arr.filter(function (strVal) { return !!strVal; });
+
+    return cssHeader() + font_face_arr.join('\n');
+  },
+
+  'css/variables': function(dictionary) {
+    var root_vars = ':root {\n' + _.map(dictionary.allProperties, function (prop) {
+      var to_ret_prop = '  --' + prop.name + ': ' + prop.value + ';';
+
+      if (prop.attributes.comment)
+        to_ret_prop = to_ret_prop.concat(' /* ' + prop.attributes.comment + ' */ ');
+      return to_ret_prop;
     }).filter(function (strVal) { return !!strVal; }).join('\n') + '\n}\n';
+
+    return cssHeader() + root_vars;
   },
 
   'scss/variables': function(dictionary) {
     var to_ret;
     return sassHeader() + _.map(dictionary.allProperties, function (prop) {
-      if (prop.attributes.category !== 'asset') {
```

Comment:
Really we should check the property values and if they don't match <declaration-value> syntax, then we should escape and quote it.  That said... for now I'll just quote things with the category of asset.

Ref: https://drafts.csswg.org/css-syntax-3/#typedef-declaration-value